R/update.r

```diff
@@ -189,28 +192,28 @@ add_versions <- function(object, to_add, archive, current) {
     add_releases_to_versions()
 }
 
-#' @importFrom falsy "%&&%"
-
 download_dcf <- function(pkg, versions, archive, current) {
   tarnames <- archive[[pkg]] %>%
     rownames()
   tarnames <- tarnames[which(ver_from_tarname(tarnames) %in% versions)]
-  url1 <- tarnames %&&% {
-    paste(sep = "/",
-          cran_site(),
-          paste(archive_path_comps, collapse = "/"),
-          tarnames)
+  
+  if (length(tarnames) > 0) {
+    url1 <- paste(sep = "/",
+                  cran_site(),
+                  paste(archive_path_comps, collapse = "/"),
+                  tarnames)
```

Comment:
@gaborcsardi above should I add tarnames inside paste as well? I think I might have missed what `&%%&` did.

Reply:
no, its good I think